src/main/java/com/ericsson/ei/frontend/BackEndInformationController.java

```diff
@@ -42,15 +42,15 @@
     @CrossOrigin
     @RequestMapping(value = "/backend", method = RequestMethod.GET)
     public ResponseEntity<String> getInstances(Model model, HttpServletRequest request) {
-        LOG.debug("Recieved request for instances.");
+        LOG.debug("Recieved request for retrieving EI Backend instances list.");
         ResponseEntity<String> response = backEndInfoContUtils.handleRequestForInstances(request);
         return response;
     }
 
     @CrossOrigin
     @RequestMapping(value = "/backend", method = RequestMethod.PUT)
```

Comment:
This method is deprecated.

Reply:
Ok, can it be removed then? Remove it in a own Pull-Request.